manifests/istio-control/istio-discovery/templates/telemetryv2_1.4.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       match:
         context: ANY # inbound, outbound, and gateway
         proxy:
-          proxyVersion: '1\.4.*'
+          proxyVersion: '^1\.4.*'
```

Comment:
side note: this still would match 1.40.1
so I guess you would really want `^1\.4(\..+)?$` instead?

Reply:
Remove trailing `.*` and add `\.` at the end of `1.4`.

Reply:
Our alpha versions are like `1.6-alpha.c091a5ab4e5ef1c2a594907949b3af096140a35b`, so that would break